device-types/Cisco/IE-4010-4S24P.yaml

```diff
@@ -0,0 +1,127 @@
+---
+manufacturer: Cisco
+model: IE-4010-4S24P
+part_number: IE-4010-4S24P
+slug: cisco-ie-4010-4s24p
+u_height: 3
+is_full_depth: false
+weight: 5.46
+weight_unit: kg
+comments: '[Industrial Ethernet 4010 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/industrial-ethernet-4010-series-switches/datasheet-c78-737279.html)'
+console-ports:
+  - name: con0
+    type: rj-45
+power-ports:
+  - name: PSU1
+    type: dc-terminal
+    allocated_draw: 70
+    maximum_draw: 250
+  - name: PSU2
+    type: dc-terminal
+    allocated_draw: 70
+    maximum_draw: 250
+interfaces:
+  - name: GigabitEthernet1/1
+    type: 1000base-t
+    poe_mode: pd
```

Comment:
`poe_mode: pd` means that this this device can be powered by poe, I think this is supposed to be `poe_mode: pse` which is for switches that can power other devices.